Cargo.toml

```diff
@@ -11,50 +11,50 @@ categories = ["command-line-utilities", "network-programming", "web-programming:
 edition = "2021"
 
 [profile.release]
+codegen-units = 1
 lto = true
 opt-level = 'z'
-codegen-units = 1
 panic = 'abort'
 
 [dependencies]
-actix-web = "4"
 actix-files = "0.6"
 actix-multipart = "0.4"
+actix-web = "4"
 actix-web-httpauth = "0.8"
-maud = "0.23"
-yansi = "0.5"
-simplelog = "0.12"
-percent-encoding = "2"
-port_check = "0.1"
-bytesize = "1"
-nanoid = "0.4"
 alphanumeric-sort = "1"
-clap = { version = "3.2", features = ["derive", "cargo", "wrap_help"] }
-clap_complete = "3.2.3"
-clap_mangen = "0.1"
+anyhow = "1"
+atty = "0.2"
+bytesize = "1"
 chrono = "0.4"
 chrono-humanize = "0.2"
-serde = { version = "1", features = ["derive"] }
-tar = "0.4"
+clap = {version = "3.2", features = ["derive", "cargo", "wrap_help"]}
+clap_complete = "3.2.3"
+clap_mangen = "0.1"
 futures = "0.3"
+get_if_addrs = "0.5"
+hex = "0.4"
+http = "0.2"
+httparse = "1"
 libflate = "1"
-thiserror = "1"
-anyhow = "1"
 log = "0.4"
-strum = "0.24"
-strum_macros = "0.24"
-sha2 = "0.10"
-hex = "0.4"
-zip = { version = "0.6.2", default-features = false }
-qrcodegen = "1"
+maud = "0.23"
 mime = "0.3"
-httparse = "1"
-http = "0.2"
-atty = "0.2"
-rustls = { version = "0.20", optional = true }
-rustls-pemfile = { version = "1.0", optional = true }
+nanoid = "0.4"
+percent-encoding = "2"
+port_check = "0.1"
+qrcode = "0.12.0"
```

Comment:
This qrcode library is technically not as well maintained but then again I think QR code libs probably don't need that much maintenance so it should be fine.

Reply:
> Can we give it a different writer?

I'll investigate

Reply:
Turns out, simply setting `stdin(Stdio::null())` fixes the problem.